modules/terraform/output.go

```diff
@@ -21,13 +20,10 @@ func Output(t testing.TestingT, options *Options, key string) string {
 
 // OutputE calls terraform output for the given variable and return its value.
 func OutputE(t testing.TestingT, options *Options, key string) (string, error) {
-	output, err := RunTerraformCommandAndGetStdoutE(t, options, "output", "-no-color", key)
+	var str string
+	err := terraform.OutputStructE(t, options, key, &str)
```

Comment:
Oh, this is a compile error. You're already in the `terraform` package, so no need to call out to it.